modules/utils/2.0/utils.smk

```diff
@@ -48,7 +48,7 @@ rule _utils_bam_sort:
     shell:
         op.as_one_line("""
         samtools sort 
-        -@ {threads} -m $(({resources.mem_mb} / {threads}))M
+        -@ {threads} -m $(({resources.mem_mb} / {threads} / 2))M
```

Comment:
Does this always return an integer even if resources.mem_mb/threads yields an odd number?

Reply:
I don't think it will always return an integer. If the threads will be odd number, this will return float, and probably exit with error. Let me try to do that  in resources, so I can convert any number to integer, and test how star runs with such modification.

Reply:
I think in @lkhilton's GRIDSS or GRIDSS-PURPLE-LINX module she had a way of doing this with Python to ensure it returns an integer

Reply:
You can use `//` in Python to drop the remainder, _i.e._ ensure an integer ([source](https://stackoverflow.com/a/1535601)).

Reply:
Thanks for all the suggestions! I don't know how it even worked in current configuration, because (48000/12/2) returns `2000.0`, and not `2000`. The `//` is good in dry run. Let me just make sure real run completes, and does not exit with any error.

Reply:
I'm using `int()` to return an integer.

Reply:
I have ensured the integer to be returned in the memory resource for the bam_sort rule in most recent commit